gravann/polyhedral/_polyhedral_labels.py

```diff
@@ -0,0 +1,58 @@
+import polyhedral_gravity
+import torch
+
+from ._polyhedral_utils import GRAVITY_CONSTANT_INVERSE, calculate_density
+
+
+def U_L(target_points, mesh_vertices, mesh_faces, density=None):
```

Comment:
I wonder if we should refactor these (also in the other labels thing) into a more verbose name? compute_potential() , compute_acceleration() ?